torch/quantization/fx/_equalize.py

```diff
@@ -436,25 +455,25 @@ def scale_weight_functional(
 
     # Scale the weights for input-weight equalization
     # If the following layer needs to be equalized then we will multiply its scale
-    scaled_weight = torch.mul(weight, torch.reciprocal(equalization_scale))
+    equalization_scale_reshaped = reshape_scale(equalization_scale, 1, weight)
+    scaled_weight = torch.mul(weight, torch.reciprocal(equalization_scale_reshaped))
 
     if next_equalization_scale is None:
         setattr(modules[weight_parent_name], weight_name, scaled_weight)
         return
 
     # Multiply the weights row wise by the next equalization scale
-    new_shape = [1] * weight.ndim
-    new_shape[0] = weight.size(0)
-    scaled_weight = torch.mul(scaled_weight, next_equalization_scale.view(new_shape))
+    next_equalization_scale_reshaped = reshape_scale(next_equalization_scale, 0, scaled_weight)
+    scaled_weight = torch.mul(scaled_weight, next_equalization_scale_reshaped)
 
     setattr(modules[weight_parent_name], weight_name, scaled_weight)
     assert(torch.allclose(model.get_buffer(str(weight_node.target)), scaled_weight))
 
     # Multiply the bias element wise by the next equalization scale
     bias_node = None
-    for node, _ in op_node.users.items():
+    for node in op_node.args:
```

Comment:
op_node.args is not the same as op_node.users, is this refactor correct?

Reply:
Yes, I realized that the original code was incorrect since the bias node should be an argument to the F.conv or F.linear layer, not a user.